external/CMakeLists.txt

```diff
@@ -42,19 +42,19 @@ endif()
 
 option(RTTR_USE_SYSTEM_LIBSAMPLERATE "Use system installed libsamplerate. Fails if not found!" "${RTTR_USE_SYSTEM_LIBS}")
 if(RTTR_USE_SYSTEM_LIBSAMPLERATE)
-	find_package(Samplerate 0.1.9 REQUIRED)
-	unset(samplerate_LIB_DEPENDS CACHE)
-	add_library(samplerate INTERFACE)
-	target_link_libraries(samplerate INTERFACE Samplerate::Samplerate)
+    find_package(Samplerate 0.1.9 REQUIRED)
+    unset(samplerate_LIB_DEPENDS CACHE)
+    add_library(samplerate INTERFACE)
+    target_link_libraries(samplerate INTERFACE Samplerate::Samplerate)
 else()
     include(FetchContent)
     FetchContent_Declare(
         LibSamplerate
-        GIT_REPOSITORY https://github.com/erikd/libsamplerate
-        GIT_TAG        727504026ee51c6fe29afe6255b70513504a6265
+        GIT_REPOSITORY https://github.com/Flamefire/libsamplerate
```

Comment:
we should put it to the common RttR location

Reply:
Why? This is a fork I maintain until my PRs are accepted into upstream. It is unrelated to Rttr and I would rather not spend even more work on that than I already have

Reply:
ah okay. It looked like we have a special adapted libsamplerate in there.

Reply:
No I've just been waiting for a year to get those PRs merged and decided to make my patches available for others to use